src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -3763,42 +3763,42 @@ class C
 ";
             var comp = CreateStandardCompilation(source, references: s_valueTupleRefs);
             comp.VerifyDiagnostics(
-                // (3,2): error CS1520: Method must have a return type
-                // {
-                Diagnostic(ErrorCode.ERR_MemberNeedsType, "").WithLocation(3, 2),
                 // (4,11): error CS1001: Identifier expected
                 //     var (x, y) = (1, 2);
-                Diagnostic(ErrorCode.ERR_IdentifierExpected, ",").WithLocation(4, 11),
```

Comment:
I didn't intend to. The helper that helps you fix this is broken. I'll add them back.